guibot/config.py

```diff
@@ -49,13 +49,10 @@ class GlobalConfig(type):
     a class object, i.e. a metaclass instance.
     """
 
-    # operational parameters shared between all instances
-    _toggle_delay = 0.05
-    _click_delay = 0.1
+    # Operational parameters shared between all instances.
```

Comment:
Let's keep this in the same style as the original comment as it doesn't relate to the changes.